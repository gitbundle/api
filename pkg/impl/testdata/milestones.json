[
  {
    "closed_at": "2020-09-11T19:32:38.046Z",
    "closed_issues": 0,
    "created_at": "2020-09-11T19:32:38.046Z",
    "description": "string",
    "due_on": "2020-09-11T19:32:38.046Z",
    "id": 1,
    "open_issues": 0,
    "state": "open",
    "title": "string",
    "updated_at": "2020-09-11T19:32:38.046Z"
  }
]
