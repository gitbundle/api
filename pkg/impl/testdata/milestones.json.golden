[
  {
    "Description": "string",
    "DueDate": "2020-09-11T19:32:38.046Z",
    "ID": 1,
    "Number": 1,
    "State": "open",
    "Title": "string"
  }
]
